lib/livebook/config.ex

```diff
@@ -291,6 +291,25 @@ defmodule Livebook.Config do
     end
   end
 
+  @doc """
+  Parses node and distribution type from env.
+  """
+  def node!(node_env, distribution_env) do
+    case {System.get_env(node_env), System.get_env(distribution_env, "sname")} do
+      {nil, _} ->
+        nil
```

Comment:
```suggestion
      {nil, _} ->
        nil

      {_, "none"} ->
        nil
```

?

Reply:
Ignore me, that's not an option for Livebook. :D

Reply:
I'm fine adding this, but didn't because it appeared that Livebook [requires distribution](https://github.com/livebook-dev/livebook/blob/25b4043dc731093358e9cb1a4f4a74639aaffe13/lib/livebook/application.ex#L86).